### DISCUSSION TITLE
Hide controls programmatically

### DISCUSSION THREAD

**Discussion by waheedakhtar694** [LunatiqueCoder/react-native-media-console#114] [Q&A]
I am changing the video size and when it's in the mini view I want to hide the controls completely, haven't found anything available for this in the docs.

**Comment by LunatiqueCoder** [comment]
@waheedakhtar694 

https://github.com/LunatiqueCoder/react-native-media-console/tree/master/packages/media-console#controls

Doesn't this work for you?
